### PR TITLE
Installation from sources draft

### PR DESCRIPTION
**Pull request by cherokee94**
@arufanov maybe we should rename "installation" to "deploy"? we don't install anything...

### CODE REVIEW

File: docs/modules/installation/pages/installation_from_sources.adoc [hunk 1]
```diff
@@ -1,3 +1,15 @@
 = Installation from sources
 
-TODO
\ No newline at end of file
+When building from source files, make sure that all required packages are installed on your system (see xref:installation_prerequisites.adoc[required software]).
```
Comment (arufanov) [inline docs/modules/installation/pages/installation_from_sources.adoc:3]: @cherokee94 
* java is not enough. See README.md file for details, but is should looks like: 

```
## Windows environment
* download and install java, at least jdk-18 (!! it is included into prereq...)
* create JAVA_HOME environment variable with value C:\Program Files\Java\jdk-19
* exit and run console again to apply environment variables
* download and install maven, at least 3.8.7
* create JAVA_HOME environment variable with value C:\apache-maven-3.8.7
* exit and run console again to apply environment variables
* download and setup nodeJS, at least 18
* install uglifycss by command: npm install -g uglifycss
* install uglifyjs by command: npm install uglify-js -gt
* setup GitHub account and add ssh keys to GitHub profile
* clone repo by command git clone: git@github.com:a5lab/axion.git
* build application by command: mvn clean package -Pdev -Dmaven.test.skip from root folder
* run application by command: mvn spring-boot:run -Pdev from root folder
* open browser with url http://127.0.0.1:8080 to view application

```


in a few works:
* install maven
* install node js
* * install uglifycss by command: npm install -g uglifycss
* install uglifyjs by command: npm install uglify-js -gt
* clone repo
* build application by command: mvn clean package -Pdev -Dmaven.test.skip from root folder
* run application by command: mvn spring-boot:run -Pdev from root folder

Also we should include infor abou 2 profiles: dev for development & production for production:

**Review by arufanov — CHANGES_REQUESTED** [review]

File: docs/modules/installation/pages/installation_from_sources.adoc (hunk 1 above)
Comment (cherokee94) [reply docs/modules/installation/pages/installation_from_sources.adoc:3]: @arufanov 

1. Updated the prerequisites with maven & uglify packages https://github.com/a5lab/axion/pull/293 
2. Need a call to discuss the profiles

Comment (cherokee94) [reply docs/modules/installation/pages/installation_from_sources.adoc:3]: @arufanov info in Installation prerequisites

**Review by arufanov — APPROVED** [review]
@cherokee94 thansk!

**Pull request merged**